code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -921,7 +921,7 @@
 		else
 			clear_fullscreen("remote_view", 0)
 
-/mob/living/silicon/ai/revive(full_heal = 0, admin_revive = 0)
+/mob/living/silicon/ai/revive()

```

Comment:
This seems backwards, all the procs should have the arguments so that keyword arguments can still be used.

Why are you doing this?